CONTRIBUTING.rst

```diff
@@ -103,11 +103,21 @@ Documentation and Comments
 Python
 ------
 
+Python Version
+~~~~~~~~~~~~~~
+
+Code should be compatible with all `supported versions <https://devguide.python.org/versions/#supported-versions>`_
+of Python.
+
 pep8
 ~~~~
 
 As a baseline, follow the `pep8 <https://www.python.org/dev/peps/pep-0008/>`_ style guide for python.
 
+We do make one exception for `maximum line length <https://peps.python.org/pep-0008/#maximum-line-length>`_.
+In Ocean we maintain a hard limit of 120 characters. It is encouraged but not required to keep lines of code
+to 100 characters or fewer and docstrings/comments to 72 characters or fewer.
```

Comment:
"docstrings/comments to 72 characters" and a few lines above it we give [an example](https://github.com/dwavesystems/dwave-ocean-sdk/blob/6b4d6cd53621e2b5f662064ba1a94dd14f93db57/CONTRIBUTING.rst?plain=1#L91) with a comment of length 96 characters